packages/react-core/src/components/Card/CardHeader.tsx

```diff
@@ -1,21 +1,53 @@
 import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/Card/card';
+import { CardContext } from './Card';
+import { Button } from '../Button';
+import AngleRightIcon from '@patternfly/react-icons/dist/js/icons/angle-right-icon';
 
 export interface CardHeaderProps extends React.HTMLProps<HTMLDivElement> {
   /** Content rendered inside the CardHeader */
   children?: React.ReactNode;
   /** Additional classes added to the CardHeader */
   className?: string;
+  /** ID of the card header */
+  id?: string;
+  /** Callback expandable card */
+  onExpand?: (cardId: string) => void;
+  /** Additional props for expandable toggle button */
+  toggleButtonProps?: any;
```

Comment:
@jessiehuff Do we need an explicit aria-label prop for the toggle button?

Reply:
I think following core's lead here would make sense. In core we have the button's `aria-label` as "Details", used in `aria-labelledby` with the card title ID, so that the button reads "{card title} details".